src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2392,7 +2392,7 @@ void CalcPurchAirLoads(EnergyPlusData &state,
                 case HumControl::Humidistat: {
                     MdotZnDehumidSP = state.dataZoneEnergyDemand->ZoneSysMoistureDemand(ControlledZoneNum).RemainingOutputReqToDehumidSP;
                     SupplyHumRatForDehum = MdotZnDehumidSP / SupplyMassFlowRate + state.dataLoopNodes->Node(ZoneNodeNum).HumRat;
-                    SupplyHumRatForDehum = min(SupplyHumRatForDehum, PurchAir(PurchAirNum).MinCoolSuppAirHumRat);
+                    SupplyHumRatForDehum = max(SupplyHumRatForDehum, PurchAir(PurchAirNum).MinCoolSuppAirHumRat);
```

Comment:
OK, so `MdotZnDehumidSP` is the kg of water per second mass flow to meet the humidity set-point.  Then we divide that by the total supply flow rate in kg of dry air per second to get the kg water per kg dry air.  Then we add the zone humidity ratio...is that right?  Maybe, I think I just need to pull out my psych chart.

![psych_chart](https://user-images.githubusercontent.com/849824/206489733-fbf9a7fa-3bef-4fd1-956c-2ae7cadb8ea0.jpg)

Anyway, then you need to take the max of that value compared to the minimum cooling supply air humidity ratio.  Yes, obviously you want to max this to constrain it to that minimum.  And since this is the only line changed, I will just move on.